package.json

```diff
@@ -31,8 +33,11 @@
 		"test": "jest --verbose --coverage"
 	},
 	"dependencies": {
+		"@microsoft/fast-components": "^2.16.3",
 		"@microsoft/fast-element": "^1.6.0",
-		"@microsoft/fast-foundation": "^2.21.0"
+		"@microsoft/fast-foundation": "^2.21.0",
+		"@microsoft/fast-react-wrapper": "^0.1.16",
+		"react": "^16.14.0"
```

Comment:
Realized this over the holiday weekend, but could we do some testing to see if it's possible to declare React as a peer dependency (or something along those lines) so the version of React that is being used by an extension author is automatically picked up/used by these components?

Basically, I would hope to avoid the scenario where someone who is using React `v17+` can't use these components because we've shipped the components using `v16`.

Reply:
I'm not entirely sure that declaring react as a peer dependency is the solution either, but seems like it might be a good starting point to explore.

Reply:
This is what `fast-react-wrapper` does as a reference point:

https://github.com/microsoft/fast/blob/master/packages/utilities/fast-react-wrapper/package.json#L91-L93

Reply:
resolved by f8be1f489161e1a9bd15a0acbed9ec09628f4ece